.github/workflows/ci-actions-incremental.yml

```diff
@@ -1079,7 +1079,7 @@ jobs:
         run: |
           git clone --depth=1 -b ${{ steps.get-quickstarts-branch.outputs.result }} https://github.com/quarkusio/quarkus-quickstarts.git && cd quarkus-quickstarts
           QUARKUS_VERSION_ARGS="-Dquarkus.platform.version=${{ steps.get-quarkus-version.outputs.quarkus-version }}"
-          export LANG=en_US && ./mvnw -e -B -fae --settings .github/mvn-settings.xml clean verify -DskipTests -Dquarkus.platform.group-id=io.quarkus $QUARKUS_VERSION_ARGS
+          export LANG=en_US && ./mvnw -e -B -fae --global-settings ../.github/mvn-settings.xml --settings .github/mvn-settings.xml clean verify -DskipTests -Dquarkus.platform.group-id=io.quarkus $QUARKUS_VERSION_ARGS
```

Comment:
I don't understand this change? I don't see how we could have both `../.github/mvn-settings.xml` and `.github/mvn-settings.xml` being right?

And from what I can see `--settings` worked until now?

Reply:
There are two sets of settings: one from the quarkus repo and one from the quickstarts.

They were identical until now.

I'm making sure both sets of settings are taken into account, because the most likely use case for this "snapshot" profile" will involve changing only the settings from the quarkus repo, to activate the `snapshots` profile, which would be ignored if we only took the settings from the quickstarts into account.

Reply:
We can, and probably should, add the `snapshots` profile to quickstarts too, but that doesn't solve the problem of activating that profile within a PR to the quarkus repo... we would only be able to change the settings from the quarkus repo, in that case.